amps/ags/rm-community/rm-community-rest-api-explorer/src/main/webapp/definitions/gs-core-api.yaml

```diff
@@ -3760,3 +3904,98 @@ definitions:
           - SiteCollaborator
           - SiteContributor
           - SiteManager
+  EventsInfo:
+    type: object
+    properties:
+      list:
+        type: object
+        properties:
+          entries:
+            type: array
+            items:
+              $ref: '#/definitions/EventInfo'
+  EventInfo:
+    type: object
+    required:
+      - id
+      - name
+      - type
+    properties:
+      id:
+        type: string
+      name:
+        type: string
+      type:
+        type: string
+  EventBodyCreate:
+    type: object
+    required:
+      - title
+    properties:
+      name:
+        type: string
+      type:
+        type: string
+        default: Simple
+        enum:
+          - Simple
+          - Obsoleted
+          - Superseded
+          - Cross-Referenced Record Transferred
+          - Versioned
```

Comment:
This should not be an enum - rather it should be an id returned by the event-types API.

Reply:
Done